nixos/modules/services/networking/networkmanager.nix

```diff
@@ -167,6 +167,18 @@ in
         '';
       };
 
+      nbft = mkOption {
```

Comment:
This doesn't hook up to `cfg.package`.

Also, you should mention that setting this option would force a rebuild of NetworkManager.

Reply:
Yeah, this doesn't really look fleshed out. We don't have any networkmanager support in initrd, so I'm not sure where this is going.

Also people can already override the package to enable nvme support, if it turns out to be a big closure size increase and we need to keep it opt-in.